main.tf

```diff
@@ -42,6 +46,32 @@ resource "aws_iam_user_policy_attachment" "policies" {
   lifecycle {
     create_before_destroy = true
   }
-  user       = join("", aws_iam_user.default.*.name)
+  user       = local.username
   policy_arn = each.value
 }
+
+module "store_write" {
+  source  = "cloudposse/ssm-parameter-store/aws"
+  version = "0.8.0"
+
+  count = module.this.enabled && var.ssm_enabled ? 1 : 0
+
+  parameter_write = [
+    {
+      name        = "/system_user/${local.username}/access_key_id"
```

Comment:
Hmm usually we expose this prefix too but we can always do that in a separate pr